exercises/practice/promises/promises.spec.js

```diff
@@ -26,12 +26,12 @@ describe('promises', () => {
     xtest("promisified function resolves to a callback's success value", () => {
       const SUCCESS = 'success';
       const fastPromise = promisify(fastCallbackFn);
-      expect(fastPromise(SUCCESS)).resolves.toEqual(SUCCESS);
+      return expect(fastPromise(SUCCESS)).resolves.toEqual(SUCCESS);
```

Comment:
The `return` is needed, see https://jestjs.io/docs/26.x/asynchronous ("Be sure to return the promise - if you omit this return statement, your test will complete before the promise returned from fetchData resolves and then() has a chance to execute the callback.")